.circleci/base_config.yml

```diff
@@ -289,28 +289,50 @@ commands:
                 git push
               fi
 
-  commit-api-docs:
-    description: Commit api-docs files to api-docs repository
-    parameters:
-      version:
-        type: string
-        default: ""
-      tag:
-        type: string
-        default: ""
+
+jobs:
+  update-vendor:
+    docker:
+      - image: cimg/go:1.21.3
```

Comment:
The latest Hugo uses 1.22.2: https://github.com/gohugoio/hugo/releases/tag/v0.125.0